client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -433,9 +432,7 @@ export class CheckoutThankYou extends React.Component {
 		}
 
 		if ( wasMarketplaceProduct ) {
-			return (
-				<AsyncLoad require="calypso/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you" />
-			);
+			return <MarketplacePluginSetupStatus />;
```

Comment:
Felt this async load is not required.

Reply:
Agreed this is useful. Checkout is a vital part of the new user experience, so the smaller we keep it, the better. It's also shared across multiple flows, so it can indefinitely increase in size if we don't take counter-measures.